bridge/hardhat/script/createRifToken.js

```diff
@@ -0,0 +1,69 @@
+// How to run the script: npx hardhat run ./hardhat/script/createRifToken.js --network bsctestnet
+const hre = require("hardhat");
+
+async function main() {
+  const {getNamedAccounts, deployments} = hre;
+  const {deployer} = await getNamedAccounts();
+
+  const transactionEtherValue = 0;
+  const tokenDecimals = 18;
+
+  const tokens = [
+    {
+      name: 'RifToken',
+      symbol: 'tRIF',
+      typeId: 1,
+      originalTokenAddress: '0x2acc95758f8b5f583470ba265eb685a8f45fc9d5',
+    }
+  ];
+
+  const Bridge = await deployments.get('Bridge');
+  const BridgeProxy = await deployments.get('BridgeProxy');
+  const MultiSigWallet = await deployments.get('MultiSigWallet');
```

Comment:
can't we get multisig and brigeProxy from the hre?

Reply:
and then here do something like `const BridgeProxy = bridgeProxy ?? await deployments.get('BridgeProxy');`

Reply:
In the `deploy` directory make sense to have the bridgeProxy from the `getNamedAccounts`, but in the scripts, it gets the addresses from the generated files in the `deployments` directory

Reply:
ohh I see, my bad